pinot-integration-tests/pom.xml

```diff
@@ -124,6 +128,74 @@
       </build>
     </profile>
   </profiles>
+  
+  <!-- cloud.localstack:localstack-utils brings in slightly older versions of all of
```

Comment:
Curious on why we need to add these dependency management to a test package. I don't think any test depends on the awssdk.
Adding @xiangfu0 to take a look

Reply:
Hi @xiangfu0 - let me know if you have any questions for me on this.

Reply:
@kkrugler Can you also comment on why you need to add these dependencies?

Reply:
I'm not adding dependencies, I'm managing the versions of all these AWS jars, since `cloud.localstack:localstack-utils` is pulling in older versions. Without this, the Maven build fails (for me) due to the dependency checker plugin that's running as part of the build.

Reply:
I think the conflict was coming from localstack, so need to manage the dependencies here